api/workspace.proto

```diff
@@ -66,9 +78,23 @@ message WorkspaceStatus {
 	string terminatedAt = 4;
 }
 
+message CreateWorkspaceBody {
+	string workspaceTemplateUid = 2;
+	int64 workspaceTemplateVersion = 3;
+
+	repeated Parameter parameters = 4;
+	repeated KeyValue labels = 5;
+}
+
 message CreateWorkspaceRequest {
 	string namespace = 1;
-	Workspace workspace = 2;
+
+	CreateWorkspaceBody body = 2;
```

Comment:
I think this should be `workspace` instead of `body`. In gRPC the request looks like this and calling the field `workspace` is more explicit:
![image](https://user-images.githubusercontent.com/1211823/81484032-45125380-91f7-11ea-8181-eac5f5176194.png)

Thoughts?

Reply:
On the other hand, looking at generated SDK, `body` makes sense in that context. I'm still testing but going to resolve this conversation.